#ifndef _APRILTAGS_MSG_HPP
#define _APRILTAGS_MSG_HPP

#include "img_capture/coord2D.h"
#include "img_capture/coord3D.h"
#include "img_capture/quat.h"
#include "img_capture/tagcenter.h"
#include "img_capture/tagcorner.h"
#include "img_capture/imgRawData.h"
#include "img_capture/pose_all.h"

//#include "img_capture/AprilTagDetection.h"
//#include "img_capture/APrilTagDetections.h"
#include "img_capture/apriltagInfo.h"
#include "img_capture/apriltagInfos.h"

#endif
